include/gatekeeper_ggu.h

```diff
@@ -47,35 +47,77 @@ struct ggu_config {
 	struct mailbox    mailbox;
 };
 
+/* Enumeration of policy decisions the GT-GK block can process. */
+enum {
+	/* Decline an IPv4 flow. */
+	GGU_DEC_IPV4_DECLINED = 0,
+	/* Decline an IPv6 flow. */
+	GGU_DEC_IPV6_DECLINED,
+	/* Grant an IPv4 flow. */
+	GGU_DEC_IPV4_GRANTED,
+	/* Grant an IPv6 flow. */
+	GGU_DEC_IPV6_GRANTED,
+	__MAX_GGU_DEC
+};
+
+/* On the wire policy decision to be processed by the GT-GK Unit. */
+struct ggu_decision {
+	/* The policy decision type. */
+	uint16_t type;
+	/* The length of the policy decision, including any parameters. */
+	uint16_t len;
```

Comment:
Do we really need the ``len`` field? Once we have the ``type`` information, we will know the exact format of the policy decision, including the length. If in the future, when we have some variable length policies, then we can introduce this field?